charts/argocd-notifications/values.yaml

```diff
@@ -87,6 +87,18 @@ secret:
       # email address in from field
       from:
 
+logLevel: info
+
+metrics:
+  enabled: false
+  port: 9001
```

Comment:
I'm not familiar with the metrics support you're adding here.  Is there anything within the project documentation to link back for those interested in exploring more?

Reply:
Thanks @andyfeller for the review. Updated description with a link to the metrics doc